lib/src/commands.dart

```diff
@@ -0,0 +1,47 @@
+import 'package:bosun/bosun.dart';
+
+class HelpCmd extends Command {
+  final Command context;
+
+  HelpCmd(this.context)
+      : super(
+            command: 'help',
+            description: 'Displays helpful information about the current project',
+            subcommands: []);
+
+  @override
+  void run(List<String> args, Map<String, dynamic> flags) {
+    String commands = '';
+    for (var element in [...context.subcommands!, ...BosunCmd(context).subcommands!]) {
+      commands += '  ${element.command}\t${element.description ?? ''}\n';
+    }
+    print('''
+A command-line utility for Dart CLI development.
+
+Usage: <command|dart-file> [arguments]
+
+Global options:
+-h, --help                 Print this usage information.
+
+Available commands:
+$commands
+Run "<command|dart-file> help" for more information about a command.
+See https://github.com/PyreStudios/bosun for detailed documentation.
```

Comment:
Similar here, we dont necessarily want to say "Hey, this CLI was built with Bosun, check out the docs here." Most of the consumers of that CLI won't really care what its built with. We can probably omit this line.

Reply:
Omitted this line.